src/api/app/models/package.rb

```diff
@@ -1408,6 +1409,13 @@ def update_activity
   def fetch_rev_from_history_cache(rev)
     Rails.cache.read(['history', self, rev]) || Rails.cache.read(['history_md5', self, rev])
   end
+
+  def populate_to_sphinx
+    if title_previously_changed? ||
```

Comment:
You can create a project without title and description. 

![Screenshot from 2021-06-22 17-13-40](https://user-images.githubusercontent.com/514785/122950996-39430b80-d37d-11eb-9628-c0b3793e653e.png)

Reply:
So true. I've fixed both Project and Package, because we can create packages without title or description as well...